services/gitclub/app/routes/helpers.py

```diff
@@ -1,24 +1,97 @@
 from os import getenv
```

Comment:
Rename this to "authorization.py" and comment it thoroughly -- we'll probably want people to follow this a bunch.